### DISCUSSION TITLE
Error 404

### DISCUSSION THREAD

**Discussion by IgnacioTosini** [community/community#33789] [New to GitHub] [answered]
Hello good evening, I have the only problem that when I want to return to the home page (the index.html) I get the error that goes through capture, is there any way to fix it?
The error is: 404
There isn't a GitHub Pages site here.

If you're trying to publish one, read the full documentation to learn how to set up GitHub Pages for your repository, organization, or user account.

**Comment by Sagar-Sharma-7** [comment]
Hello @IgnacioTosini 
You can publish your HTML project by following the steps below:
1) Click on the setting of the repo you want to publish.
2) On the left side, click on pages and then select "Deploy from branch" and then select branch to "main" of any branch which contains your HTML file.
3) click on save and wait up to 3 mins and then refresh, you going to see a link there, click on it.

**Comment by DSDmark** [comment]
> Hello good evening, I have the only problem that when I want to return to the home page (the index.html) I get the error that goes through capture, is there any way to fix it? The error is: 404 There isn't a GitHub Pages site here.
> 
> If you're trying to publish one, read the full documentation to learn how to set up GitHub Pages for your repository, organization, or user account.

Can you add some more details like link or maybe your repo info something's .
It's maybe possible you mention wrong path in your files when you go in home page

**Reply by IgnacioTosini** [reply]
https://github.com/IgnacioTosini/CoderCurso this is my repository, the issue is that now I put the html files in folders without having solved the previous problem.

**Reply by DSDmark** [reply]
@IgnacioTosini so if you find the answer as @james-villanueva-freee describe then close this question by click on accepting the **answer button**

**Comment by james-villanueva-freee** [comment, marked as answer]
Hi. I looked at your code and it seems like your `Inicio` links have the wrong `href` values. Since those links point to `index.html` which is outside the `pages` folder, you want it to go out of the folder first then find the `index.html`. Like this:

`<li><a href="../index.html">Inicio</a></li>`

Kindly apply the new `href` to all `Inicio` links inside the pages folder. I hope that fixes it 🙇🏼‍♂️
